package.json

```
@@ -16,7 +16,9 @@
     "test:demo": "ng test demo --browsers ChromeHeadless --watch false",
     "test:lib": "ng test ngx-editor --browsers ChromeHeadless --watch false",
     "test": "npm run test:lib && npm run test:demo",
-    "prepare": "sh ./scripts/install-husky.sh"
+    "prepare": "run-script-os",
```

```bash
npm i -D is-ci
```

we can install `is-ci`, which is already part of the dependency tree 

```suggestion
    "prepare": "is-ci || husky install",
```
we don't need `run-script-os`